openedx/features/course_card/views.py

```diff
@@ -66,8 +69,16 @@ def get_course_cards(request):
     )
 
 
-def get_course_with_link_and_start_date(course, course_rerun_object, request):
+def get_course_dict(course, programs):
```

Comment:
How about to move this method in helpers?

Reply:
its for course data population and gives code reader a better readability. Moreover, I don't think it falls under the category of helpers because the implementation is explicitly for this specific view. thoughts?

Reply:
I didn't get your readability point, Actually this is a method which takes a course and a program and tells either this course is part of the program or not it's being used in view but not a view.
Let's hop on a call if still there is any confusion.